core/runtime/src/main/java/io/quarkus/runtime/configuration/DurationConverter.java

```diff
@@ -32,6 +32,9 @@ public DurationConverter() {
      */
     @Override
     public Duration convert(String value) {
+        if (value.isEmpty()) {
```

Comment:
Should we trim this?

Reply:
Yes, that's a good idea.  This change predates our big discussion.  I'll add it.